dist-unminified/recorder.js

```diff
@@ -100,6 +100,7 @@ var Recorder = function( config ){
     leaveStreamOpen: false,
     maxBuffersPerPage: 40,
     monitorGain: 0,
+    microphoneGain: 0.9,
```

Comment:
Why default to 0.9? I would prefer not to change the default behaviour for users that choose not to use this feature.

Reply:
updated default to 1